syntheseus/tests/interface/test_models.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+import numpy as np
+
+from syntheseus.interface.models import Prediction
+
+
+def test_prediction():
```

Comment:
Is this test just there to make sure imports from `models.py` work? If so, maybe add this as a comment or TODO item in the code to remind our future selves 😛